cmd/wwclient/wwclient.go

```diff
@@ -67,14 +69,38 @@ func main() {
 		},
 	}
 
+	smbiosDump, err := smbios.New()
+	if err != nil {
+		wwlog.Printf(wwlog.ERROR, "Could not get SMBIOS info: %s\n", err)
+		os.Exit(1)
+	}
+	sysinfoDump := smbiosDump.SystemInformation()
+	localUUID, _ := sysinfoDump.UUID()
+	x := smbiosDump.SystemEnclosure()
+	tag := x.AssetTagNumber()
+
+	cmdline, err := ioutil.ReadFile("/proc/cmdline")
+	if err != nil {
+		wwlog.Printf(wwlog.ERROR, "Could not read from /proc/cmdline: %s\n", err)
+		os.Exit(1)
+	}
+
+	wwid_tmp := strings.Split(string(cmdline), "wwid=")
+	if len(wwid_tmp) < 2 {
+		wwlog.Printf(wwlog.ERROR, "'wwid' is not defined in /proc/cmdline\n")
+		os.Exit(1)
+	}
+
+	wwid := strings.Split(wwid_tmp[1], " ")[0]
+
 	for {
 		var resp *http.Response
 		counter := 0
 
 		for {
 			var err error
 
-			getString := fmt.Sprintf("http://%s:%d/overlay-runtime", conf.Ipaddr, conf.Warewulf.Port)
+			getString := fmt.Sprintf("http://%s:%d/overlay-runtime/%s?assetkey=%s&uuid=%s", conf.Ipaddr, conf.Warewulf.Port, wwid, tag, localUUID)
```

Comment:
in line 88ff. `wwid` is parsed as a generic variable. It is only now used exclusively for the Hwaddr, as the daemon is setting it that way. This might change in the future.
Wouldn't it be prudent to explicitly declare it via `?wwid=%s` in `getString`?

Reply:
I'm not sure I follow here... I'm going to merge this as it fixes a issue in current main with provisioning the runtime overlay. Can you open this as an issue?